### ISSUE TITLE
Disabling svg breaks mods page

### ISSUE THREAD

**Issue by SFort** [modrinth/knossos#385]
### Environment

Website

### Describe the bug

The [mods](https://modrinth.com/mods) search bar, page and sort by buttons, depend on svg being enabled / the browser supporting svg in order to function

the rest of the website is perfectly functionaly without svg support

### Steps To Reproduce

- open firefox
- goto about:config
- set svg.disabled to true

**Comment by Prospector** [comment]
Looking into this, I cannot reproduce it in dev, but I can on both https://modrinth.com/mods and https://staging.modrinth.com/mods

In addition, no errors seem to be put in the console or anything, so I am at a loss as to what the problem could be. It's worth noting the sveltekit rewrite seems to work flawlessly with SVG disabled.

**Comment by SFort** [comment]
> is there any reason why you want to disable svgs?

personaly i could turn it on, just so happens that a anti-fingerprinting file i imported has them disabled.

having a website stop functioning because an image didn't load is silly and if you care about comatability / accessability it's bad practice to depend on what are advanced browser features  for functionality which dosn't even use those features.

**Comment by wafflecoffee** [comment]
As far as I can tell, this is entirely due to the LogoAnimated svg which is there while the search data loads

**Comment by lucsoft** [comment]
![image](https://user-images.githubusercontent.com/11093382/156931471-31ed7c03-4c06-478c-8913-5800050944da.png)
Error Code `TypeError: can't access property "fill", t.style is undefined`

This happens while the hydration is going and is due the api is missing not that svg didn't load

> comatability
> advanced browser features

we are talking about svg, that's even before chrome existed.

**Comment by SFort** [comment]
fair "advanced browser features" isn't accurate.
there are browsers and accessibility tools that can't read svgs (even if extreamly uncommon).
functionality shouldn't break for aesthetic

**Comment by triphora** [comment]
Going to mark this as not planned. It simply isn't worth the amount of effort it would take to make the site work with SVG disabled, and this is the only report we've ever received of anyone browsing without SVG.
